modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -267,29 +297,28 @@ def location(type, appointment_hash)
             end
             location[:lat] = appointment_hash.dig(:location, :lat)
             location[:long] = appointment_hash.dig(:location, :long)
-            location[:phone] = location_phone(appointment_hash)
+            location[:phone] = parse_phone(appointment_hash.dig(:location, :phone, :main))
           end
 
           location
         end
         # rubocop:enable Metrics/MethodLength
 
-        def location_phone(appointment_hash)
-          phone = appointment_hash.dig(:location, :phone, :main)
-
+        def parse_phone(phone)
           # captures area code (\d{3}) number (\d{3}-\d{4})
           # and optional extension (until the end of the string) (?:\sx(\d*))?$
-          phone_captures = phone&.match(/^(\d{3})-(\d{3}-\d{4})(?:\sx(\d*))?$/)
+          phone_captures = phone&.match(/^\(?(\d{3})\)?.?(\d{3}-?\d{4})(?:\sx(\d*))?$/).to_a
```

Comment:
was necessary due to the regex matcher returning a "matchdata" object that didn't allow you to change the values. but that isn't necessary anymore since the dash is being added to the string rather then the capture groups.